python/ray/tune/tune.py

```diff
@@ -30,6 +30,9 @@ def _make_scheduler(args):
 
 
 def run_experiments(experiments, scheduler=None):
+    # Make sure rllib agents are registered
+    from ray import rllib  # noqa # pylint: disable=unused-import
```

Comment:
Importing rllib actually does not have any hard dependencies -- if some dependency is not installed you just get a warning on the console.

Reply:
OK, I see. I'm trying to import `run_experiments` on a clean env.

I'm getting an import error telling me that I still need to install TensorFlow (for the Logger) - this should be documented somewhere. Could also just disable the TensorBoard logger if Tensorflow not installed. This can be done on a separate PR (or in the current one)

Also, if a user is trying to use `tune` out of the box (say for HyperBand), it probably makes sense to preface the warnings with something like `Trying to import default rllib agents...` instead of a sudden block of warnings.

Reply:
That's fine for now, we can improve this in a separate PR.